src/Sulu/Bundle/AdminBundle/Controller/AdminController.php

```diff
@@ -208,7 +209,28 @@ public function contextsAction(Request $request)
         $contexts = $this->adminPool->getSecurityContexts();
         $system = $request->get('system');
 
-        $response = isset($system) ? $contexts[$system] : $contexts;
+        $mappedContexts = [];
+
+        foreach ($contexts as $system => $sections) {
+            foreach ($sections as $section => $contexts) {
+                foreach ($contexts as $context => $permissionTypes) {
+                    if (is_array($permissionTypes)) {
+                        $mappedContexts[$system][$section][$context] = $permissionTypes;
+                    } elseif (is_string($permissionTypes)) {
+                        // FIXME for keeping BC, system used to only have context names instead of permission types
+                        $mappedContexts[$system][$section][$permissionTypes] = [
+                            PermissionTypes::VIEW,
```

Comment:
For me it is hard to say what are the defaults... Some contexts have a security permission, some don't. That's why I would try to avoid having predefined defaults at all...